unicode-data-names/unicode-data-names.cabal

```diff
@@ -85,6 +90,36 @@ library
   hs-source-dirs: lib
   build-depends:
       base             >= 4.7   && < 4.19,
+  -- Support for raw string literals unpacking is included in base ≥ 4.15
+  if impl(ghc < 9.0.0)
+    build-depends:
+      ghc-prim >= 0.3.1 && < 1.0
+
+-- Internal library to enable comparison to ICU
+library icu-names
+  import: default-extensions, compile-options
+  if flag(has-icu)
+    buildable: True
+  else
+    buildable: False
+  exposed-modules:
+    ICU.Names
+  hs-source-dirs: lib
+  build-depends:
+      base             >= 4.7   && < 4.19
+  include-dirs: cbits
+  c-sources: cbits/icu.c
+  cc-options: -Wall -Wextra -pedantic
```

Comment:
Should also use optimization option, `-O2` should be fine or maybe `-Ofast`.

Reply:
It is not mandatory, it will be added with e.g. `cabal -O2 run`.